bin/compile

```diff
@@ -175,6 +190,7 @@ chmod +x "$BUILD_DIR/bin/jq"
 
 DEFAULT_PHP=$(curl --fail --location --silent "${SEMVER_SERVER}/php-${STACK}")
 DEFAULT_NGINX=$(curl --fail --location --silent "${SEMVER_SERVER}/nginx-${STACK}")
+DEFAULT_COMPOSER=$(curl --fail --location --silent "${SEMVER_SERVER}/composer-${STACK}")
```

Comment:
What does this return? It should be `1` today no? We don't want to use `2` by default, maybe we'll change like in N months

Reply:
I defaulted to the 2.x (https://semver.scalingo.com/composer-scalingo-18/). But I can change it in semver if you prefer. The Composer announcement of the 2.0 (https://blog.packagist.com/composer-2-0-is-now-available/) highlight that everyone should switch to Composer 2 ASAP:

> As for Composer 1.x, it is now more or less EOL. It will also receive critical fixes if anything comes up but the goal for everyone should be to migrate to 2.x as soon as possible.

Reply:
Yes but we've seen it's breaking builds, so it's quite annoying

Reply:
They release one day composer 2.0.0 and announce EOL of 1.x 2 days later? Terrible logics

Reply:
Yep definitely. But we can keep the v1 for a couple of months and I add a reminder to think about making the switch in a couple of months if you think it's ebtter

Reply:
A warning line in the build to start?

Reply:
I think if we can prevent breaking builds, it will be less support ;-)

Reply:
yep sure, you're definitely right (as usual...)!

Reply:
With a `sleep` of 10 seconds to make sure it's read by the customers? :p

Reply:
Not necessarily a sleep

Reply:
but a large "warning" Blinking and everything

Reply:
The default is now 1.x https://github.com/Scalingo/semver.io/pull/10

i'll update the buildpack to add the warning